src/IW/App/Error.hs

```diff
@@ -0,0 +1,193 @@
+{-# LANGUAGE ConstraintKinds #-}
```

Comment:
`ConstraintKinds` can be put into `default-extensions`

Reply:
Got it